easy-ui-react/src/EmptyStateCard/EmptyStateCard.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { ReactNode, useMemo } from "react";
+import { Text, TextProps } from "../Text";
+import { Card } from "../Card";
+import { VerticalStack } from "../VerticalStack";
+import { flattenChildren } from "../utilities/react";
+import { ResponsiveSpaceScale } from "../types";
+
+export type EmptyStateCardProps = {
+  /**
+   * The children of the <EmptyStateCard> element. Should render
+   * `<EmptyStateCard.Header>`, `<EmptyStateCard.Body>`, and
+   * `<EmptyStateCard.Action>` at minimum.
+   */
+  children: ReactNode;
+  /**
+   * Gap between `<EmptyStateCard.Header>` and `<EmptyStateCard.Body>`
+   * @default 2
+   */
+  primaryGap?: ResponsiveSpaceScale;
+  /**
+   * Gap between `<EmptyStateCard.Header>` and `<EmptyStateCard.Body>`
+   * block and `<EmptyStateCard.Action>`
+   * @default 2
+   */
+  secondaryGap?: ResponsiveSpaceScale;
```

Comment:
looking at the Figma, it looks as though we need to support a variation where the gap is different

Reply:
Suggestion: what do you think of `textGap` for the gap between header and body and `blockGap` for the gap between cta and text?

Reply:
good call, i like that better